app/assets/javascripts/misc/toggle-button.js

```diff
@@ -0,0 +1,24 @@
+import 'classlist.js';
+
+function toggleButton() {
+  const accordion = document.querySelector('.accordion');
+
+  if (accordion) {
+    const controls = accordion.querySelectorAll('[aria-controls]');
+    const button = document.querySelector('.js-toggle-button');
+
+    controls.forEach((control) => {
```

Comment:
doesn't `querySelectorAll` return a NodeList which doesn't have `forEach`? I'd expect we have to `forEach.apply(...` here